### PR TITLE
Spark | Update getting started tutorial

### LINKED ISSUE #19668: microsoft-spark-2.4.x-<version>.jar
[Enter feedback here]
Where is microsoft-spark-2.4.x-<version>.jar from? I do not see it in this guide.
Thank you.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0f1dc518-dfe4-b71a-ee04-e8520bbd2f67
* Version Independent ID: e04b082c-e52b-3ffb-c69d-10da5dc54528
* Content: [Get started with .NET for Apache Spark](https://docs.microsoft.com/en-gb/dotnet/spark/tutorials/get-started)
* Content Source: [docs/spark/tutorials/get-started.md](https://github.com/dotnet/docs/blob/master/docs/spark/tutorials/get-started.md)
* Product: **dotnet-spark**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by luisquintanilla**
## Summary

- Updates to get started tutorial code and version used
- Clarify where files created at build time reside.

Fixes #19668 

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/spark/tutorials/get-started?branch=pr-en-us-20264&tabs=windows#write-a-net-for-apache-spark-app)

### CODE REVIEW

File: docs/spark/tutorials/get-started.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 title: Get started with .NET for Apache Spark
 description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows, MacOS, and Ubuntu.
-ms.date: 06/25/2020
+ms.date: 08/24/2020
 ms.topic: tutorial
 ms.custom: mvc
+ms.author: luquinta
+author: luisquintanilla
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:8]: These are already set globally in docfx.json.

```suggestion
```

File: docs/spark/tutorials/get-started.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 ---
 title: Get started with .NET for Apache Spark
 description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows, MacOS, and Ubuntu.
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:3]:
```suggestion
description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows, macOS, and Ubuntu.
```

File: docs/spark/tutorials/get-started.md [hunk 3]
```diff
@@ -142,43 +144,49 @@ The `dotnet` command creates a `new` application of type `console` for you. The
 
 To use .NET for Apache Spark in an app, install the Microsoft.Spark package. In your command prompt or terminal, run the following command:
 
-`dotnet add package Microsoft.Spark --version 0.8.0`
+`dotnet add package Microsoft.Spark`
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:147]:
~~~suggestion
```dotnet
dotnet add package Microsoft.Spark
```
~~~

File: docs/spark/tutorials/get-started.md [hunk 4]
```diff
@@ -142,43 +144,49 @@ The `dotnet` command creates a `new` application of type `console` for you. The
...
 ### 3. Code your app
 
 Open *Program.cs* in Visual Studio Code, or any text editor, and replace all of the code with the following:
 
 ```csharp
 using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
 
-namespace MySparkApp
+namespace mySparkApp
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:160]: I believe the convention for naming namespaces is PascalCase. Why this is changed to camelCase?

File: docs/spark/tutorials/get-started.md [hunk 5]
```diff
@@ -142,43 +144,49 @@ The `dotnet` command creates a `new` application of type `console` for you. The
...
         {
-            // Create a Spark session.
-            SparkSession spark = SparkSession
-                .Builder()
-                .AppName("word_count_sample")
-                .GetOrCreate();
-
-            // Create initial DataFrame.
+            // Create spark session
+            SparkSession spark =
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:167]:
```suggestion
            SparkSession spark =
```

File: docs/spark/tutorials/get-started.md [hunk 6]
```diff
@@ -201,6 +209,18 @@ This .NET app uses .NET for Apache Spark
 This .NET app counts words with Apache Spark
 ```
 
+  Save the changes.
+
+Open the *mySparkApp.csproj* file. Copy the newly created *input.txt* file to your build output directory by adding the following contents inside the `Project` tags:
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:214]:
```suggestion
Open the *mySparkApp.csproj* file. Copy the newly created *input.txt* file to your build output directory by adding the following contents inside the `Project` tag:
```

File: docs/spark/tutorials/get-started.md [hunk 7]
```diff
@@ -201,6 +209,18 @@ This .NET app uses .NET for Apache Spark
...
+
+ ```xml
+<ItemGroup>
+    <Content Include="input.txt">
+    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+ </ItemGroup>
```
Comment (Youssef1313) [inline docs/spark/tutorials/get-started.md:221]:
```suggestion
<ItemGroup>
  <Content Include="input.txt">
    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
  </Content>
</ItemGroup>
```

File: docs/spark/tutorials/get-started.md (hunk 2 above)
Comment (adegeo) [reply docs/spark/tutorials/get-started.md:3]: Also, is there a reason Ubuntu is called out directly? Should this just be Linux?

File: docs/spark/tutorials/get-started.md [hunk 8]
```diff
@@ -209,14 +229,14 @@ This .NET app counts words with Apache Spark
    dotnet build
    ```
 
-2. Run the following command to submit your application to run on Apache Spark:
+2. Navigate to your build output directory and enter the following command to submit your application to run on Apache Spark:
```
Comment (adegeo) [inline docs/spark/tutorials/get-started.md:232]: The example below is using bash style line continuations. This code doesn't copy\paste to windows unless someone is using a bash terminal. This doesn't work on powershell or cmd.exe

**Review by adegeo — DISMISSED** [review]
I believe that @Youssef1313 covered some of the fixes. There is another lint violation to take care of an a few comments from me. Cheers!

**Comment by luisquintanilla** [comment]
Thanks @Youssef1313 and @adegeo for the review. I'll take a look and make the changes

File: docs/spark/tutorials/get-started.md [hunk 9]
```diff
@@ -142,43 +144,49 @@ The `dotnet` command creates a `new` application of type `console` for you. The
...
-                .Select(Functions.Split(Functions.Col("value"), " ").Alias("words"))
-                .Select(Functions.Explode(Functions.Col("words"))
-                .Alias("word"))
-                .GroupBy("word")
-                .Count()
-                .OrderBy(Functions.Col("count").Desc());
-
-            // Show results.
+            // Count words
+            DataFrame words =
```
Comment (JohyPark) [inline docs/spark/tutorials/get-started.md:177]:
```suggestion
            DataFrame words =
```

**Review by johypark — CHANGES_REQUESTED** [review]

**Comment by adegeo** [comment]
@luisquintanilla ping

**Comment by luisquintanilla** [comment]
@adegeo other Ignite related work items took priority but planning to merge this week.

File: docs/spark/tutorials/get-started.md (hunk 2 above)
Comment (luisquintanilla) [reply docs/spark/tutorials/get-started.md:3]: The .NET for Spark worker package targets the Ubuntu distro.

**Comment by adegeo** [comment]
@luisquintanilla No rush! Just friendly ping 😄

**Comment by luisquintanilla** [comment]
Okay. I think this one's good to go. Whenever you get a chance @adegeo. Thank you! :)

**Comment by luisquintanilla** [comment]
@mamccrea can I please get a review on this when you get a chance. Thanks 🙂

**Review by mamccrea — APPROVED** [review]
LGTM

**Pull request merged**